gammapy/background/tests/test_fov.py

```diff
@@ -1,2 +1,110 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import print_function, division
+
+from gammapy.background import fill_acceptance_image
+from gammapy.image import make_empty_image
+
+from astropy.io import fits
+from astropy.coordinates import SkyCoord, Angle
+from astropy.wcs import WCS
+from astropy.wcs.utils import pixel_to_skycoord
+import astropy.units as u
+from astropy.units.quantity import Quantity
+
+import numpy as np
+from scipy.integrate import quad
+
+def radial_gaussian_2D(r, sig):
```

Comment:
I think this function is superfluous and should be removed.
Either call `astropy.modeling.models.Gaussian1D(offset)` or `astropy.modeling.models.Gaussian2D(offset, 0)` (I didn't look in detail what you implement here or need).

If you absolutely want a model that takes `Angle` objects as input and the Astropy models don't work the way you want for some reason, OK to write a wrapper, but internally please call the Astropy model and don't re-implement the formula. (But I think it's not needed, especially if you only use this in a test.)